### PR TITLE
Improve kops edit command

### LINKED ISSUE #16104: Improvement in ```kops edit``` command
image
```kops edit cluster``` command does not give a good description of how it can be edited.
Here it says it directly sets values in the spec.
But to set values in the spec you can not use direct value assignment:
For example, following one does not works:
```
kops edit cluster testcluster.k8s.local --set kubernetesVersion=1.28.4 
```
It has to be ```spec.kubernetesVersion```:
```
kops edit cluster testcluster.k8s.local --set spec.kubernetesVersion=1.28.4 
```

Either we can specify that in the readme.

OR

We can add a example.

### PR DESCRIPTION
**Pull request by prashantrewar**
Fixes #16104

### CODE REVIEW

**Comment by hakman** [comment]
Hi @prashantrewar, this change should be done in https://github.com/kubernetes/kops/blob/master/cmd/kops/edit_cluster.go.
Docs are updated by running `make gen-cli-docs`.

**Comment by prashantrewar** [comment]
Thank you @hakman. I have made the changes.

File: cmd/kops/edit_cluster.go [hunk 1]
```diff
@@ -66,6 +66,9 @@ var (
 	editClusterExample = templates.Examples(i18n.T(`
 	# Edit a cluster configuration in AWS.
 	kops edit cluster k8s.cluster.site --state=s3://my-state-store
+
+	# Set Values in the spec.
```
Comment (hakman) [inline cmd/kops/edit_cluster.go:70]:
```suggestion
	# Set cluster spec values.
```

File: docs/cli/kops_edit_cluster.md [hunk 2]
```diff
@@ -25,6 +25,9 @@ kops edit cluster [CLUSTER] [flags]
 ```
   # Edit a cluster configuration in AWS.
   kops edit cluster k8s.cluster.site --state=s3://my-state-store
+  
+  # Set Values in the spec.
```
Comment (hakman) [inline docs/cli/kops_edit_cluster.md:29]:
```suggestion
  # Set cluster spec values.
```

**Review by hakman — CHANGES_REQUESTED** [review]

**Comment by hakman** [comment]
Thanks @prashantrewar! :)
/lgtm
/approve

**Pull request merged**